### ISSUE TITLE
Directive completion for script commands

### ISSUE THREAD

**Issue by pomdtr** [tonka3000/vscode-raycast#12]
It would be cool to add some validation / completions for the `@raycast` directives in script commands.

Not sure how to achieve this though, since the scripts commands can be written in any language.

**Comment by pomdtr** [comment]
Global snippets might do the trick:

```
{
	"raycast.schemaVersion": {
		"prefix": "@raycast.schemaVersion",
		"body": "$LINE_COMMENT @raycast.schemaVersion 1"
	},
	"raycast.mode": {
		"prefix": "@raycast.mode",
		"body": "$LINE_COMMENT @raycast.mode ${1|silent,compact,inline,fullOutput|}"
	},
	"raycast.argument": {
		"prefix": "@raycast.argument",
		"body": "$LINE_COMMENT @raycast.argument${1|1,2,3|} { \"type\": \"text\", \"placeholder\": \"${3:Placeholder}\", \"optional\": ${4|true,false|} }"
	},
}
```

**Comment by tonka3000** [comment]
Hi @pomdtr ,

Great idea. I've added all the snippet in #14.

example:

https://user-images.githubusercontent.com/3163807/181909244-20c120ea-9dd0-4dfb-9e31-18ee08590a61.mp4

I could add this to all files, but that would make much sense (e.g. css). I've added it now for 

- ruby
- python
- swift
- shellscript
- typescript
- javascript
- applescript

This are all languages which are used in the [scripts-commands repo](https://github.com/raycast/script-commands).

I've excluded plaintext for now, because $LINE_COMMENT is not set in the language.

Greetings
Michael
